examples/textual_inversion/textual_inversion.py

```diff
@@ -48,6 +49,10 @@
 # ------------------------------------------------------------------------------
 
 
+# Will error if the minimal version of diffusers is not installed. Remove at your own risks.
+check_min_version("0.10.0.dev0")
```

Comment:
Only annoyance is we might have to manually bump these every so often. I'd generally be ok with letting the warning in the readme serve as the primary reminder with maybe an additional comment in the scripts just to minimize manual work. No strong feelings however